src/clients/python/sams_client/client.py

```diff
@@ -95,6 +95,14 @@ def request(
         base_url = self.config.get('base_url')
         url = f'{base_url}{api}'
         headers = self.auth.apply_headers(headers)
+
+        if isinstance(data, dict):
+            tags = data.get('tags')
+            if tags is not None:
+                if len(tags) >= 0:
+                    headers.update({'Content-Type': 'application/json'})
```

Comment:
This API consumes all `POST` and `PATCH` requests as `application/json`. Shouldn't this header by applied for all those types to enforce this?